modules/swagger-codegen/src/main/resources/Java/libraries/retrofit2/build.gradle.mustache

```diff
@@ -94,9 +94,8 @@ if(hasProperty('target') && target == 'android') {
 }
 
 ext {
-    oltu_version = "1.0.1"
-    retrofit_version = "2.0.0-beta4"
-    gson_version = "2.6.2"
+    oltu_version = "1.0.0"
```

Comment:
Will do asap.

On Wed, Apr 27, 2016 at 8:14 AM, wing328 <notifications@github.com> wrote:

> In
> modules/swagger-codegen/src/main/resources/Java/libraries/retrofit2/build.gradle.mustache
> <https://github.com/swagger-api/swagger-codegen/pull/2479#discussion_r61208586>
> :
>
> > @@ -94,9 +94,8 @@ if(hasProperty('target') && target == 'android') {
> >  }
> >
> >  ext {
> > -    oltu_version = "1.0.1"
> > -    retrofit_version = "2.0.0-beta4"
> > -    gson_version = "2.6.2"
> > +    oltu_version = "1.0.0"
>
> Yes please. We want to upgrade the dependencies to the latest version
> after stable releases.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/swagger-api/swagger-codegen/pull/2479/files/f528ee62c911a26f860dbc1d7878ee87a0f4778b#r61208586>
>